docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1975 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.        P[~T_subseq_isfinite] = np.NINF #to get finite-values discords
</pre><p>All subsequences that have non-finite values will automatically have <code>P</code>   values that are <code>np.inf</code> .  In STUMPY, it is impossible for a subsequence to have non-finite values and for it's matrix profile value to be finite!</p><p> </p><p>So you should change this to <code>P[~np.isfinite[P]] = np.NINF</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
I see! one line less! Will revise it.